products/rtpengine.md

```diff
@@ -0,0 +1,40 @@
+---
+title: rtpengine 
+category: server-app
+permalink: /rtpengine
+versionCommand: rtpengine -v
+releaseDateColumn: true
+
+identifiers:
+-   repology: rtpengine
+
+releases:
+
+-   releaseCycle: "mr13.0"
+    releaseDate: 2024-10-01
+    eol: false
+    latest: "mr13.0.1"  
+
+-   releaseCycle: "mr12.5"
+    releaseDate: 2024-08-23
+    eol: false
+    latest: "mr12.5.1"
+
+-   releaseCycle: "mr11.5"
+    releaseDate: 2023-09-22
+    eol: false
+    latest: "mr11.5.1"
+
+-   releaseCycle: "mr10.5"
+    releaseDate: 2022-07-08
+    lts: true
+    eol: false
+    latest: "mr10.5.7"
+    latestReleaseDate: 2024-05-10
+
+---
+
+> The [Sipwise NGCP rtpengine](https://www.sipwise.com/products/rtpengine/) is a proxy for RTP traffic and other UDP based media traffic. 
+> It’s meant to be used with the Kamailio SIP proxy and forms a drop-in replacement for any of the other available RTP and media proxies.
+
+
```

Comment:
Could you elaborate on the release, support and EOL policy:

- How frequent are the releases ?
- For how long new releases are supported ?
- What is the level of support for each releases (only bug / security fixes, also contains new features...) ?
- Is it documented somewhere on https://www.sipwise.com ?
- ...

Reply:
will dig into this, didn't find any public information about this. they just communicate a release map : https://www.sipwise.com/releases/

Reply:
Would be great to document this. Something like:

```
rtpengine release, support and EOL policies are not documented, but support status for each release can be found on <https://www.sipwise.com/releases/>.
```

Reply:
fixed